costar_dataset/block_stacking_reader_torch.py

```diff
@@ -616,6 +622,68 @@ def __init__(self, list_example_filenames,
         if self.inference_mode is True:
             self.list_example_filenames = inference_mode_gen(self.list_example_filenames)
 
+    @classmethod
+    def from_parameters(cls, root, version, set_name, subset_name, split, feature_mode=None,
+                        total_actions_available=41,
+                        seed=0, random_state=None,
+                        is_training=True, random_augmentation=None,
+                        random_shift=False,
+                        output_shape=None,
+                        blend_previous_goal_images=False,
+                        estimated_time_steps_per_example=250, verbose=0, inference_mode=False, one_hot_encoding=True,
+                        pose_name='pose_gripper_center',
+                        force_random_training_pose_augmentation=None):
+        '''Call constructor from specified parameter
```

Comment:
Can you add a proper docstring to this? What is it for, what do you integrate it with, why should you call it? Where can I find a useful example?

Reply:
Also, what are the options for stuff like feature_mode

Reply:
Detailed docstring added.